.gitlab-ci.yml

```diff
@@ -34,6 +34,10 @@ build:
   script:
     - make deps
     - make build
+  artifacts:
+    paths:
+      - dist/
+    when: always
```

Comment:
Why `always`? Will there be something there to help debugging if building the `dist` fails?

Same for the later uses of `when: always`.

Reply:
Yes. In case the job fails at the deploy level but we still want to create a release manually, artifacts should still be stored.